go/src/staleFlagCleaner.go

```diff
@@ -0,0 +1,624 @@
+/*
+Copyright (c) 2021 Uber Technologies, Inc.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+
+except in compliance with the License. You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the
+
+License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied. See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package src
+
+import (
+	"encoding/json"
+	"go/token"
+	"io/ioutil"
+	"log"
+	"os"
+
+	/*
+		We have used dst package instead of official go ast package because ast package
+		does not keep positions of comment in place. So, we use dst package which keeps
+		the positions of comment in place. It is similiar to ast package and have same
+		functions as ast. So you can use documentation of ast to see the details of helper
+		functions.
+	*/
+	"github.com/dave/dst"
+	"github.com/dave/dst/dstutil"
+)
+
+// API : For the type of API
+type API int
+
+const (
+	isTreated API = iota
+	isControl
+	isTesting
+	isUnknown
+)
+
+// Methods : For json it is
+type Methods struct {
+	Methods []FlagAPI `json:"methodProperties"`
+}
+
+// FlagAPI : for getting the type from config file
+type FlagAPI struct {
+	MethodName string `json:"methodName"`
+	OfType     string `json:"flagType"`
+	FlagIndex  int    `json:"argumentIndex"`
+}
+
+// Value : This is made to not confuse with true and false in code
+type Value int
+
+const (
+	isTrue Value = iota
+	isFalse
+	// This will be returned when you don't have to find true or false
+	isBot
```

Comment:
This name choice is a bit weird, unless it really has something to do with some "bot". Otherwise, why don't you use something like `isUndefined`?

Reply:
It comes from the usage in lattices. https://www.cs.yale.edu/homes/wilke-pierre/itp17/doc/html/Lattice.html. 
But I agree that isUndefined is more descriptive :)

Reply:
I agree. I will change.